calc_app/aggregations.py

```diff
@@ -0,0 +1,8 @@
+from .models import ExpenseItem, Category
+
+
+def get_categories_costs(names):
+    name_dict = {name: 0 for name in names}
+    for item in ExpenseItem.objects.all().select_related("category"):
```

Comment:
as far as I understand, `ExpenseItem.objects.select_related('category')` does the same thing

Reply:
This string joins two tables using ORM to prevent N+1 problem and optimizes the request to the database. What thing you think it does?

Reply:
Updated